src/RepecId.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace Altmetric\Identifiers;
+
+class RepecId
+{
+    public static function extract($str)
+    {
+        preg_match_all('/\brepec:\S+\b/i', $str, $matches);
+
+        return array_map(function($repecId) {
```

Comment:
As the matches are guaranteed to have the `repec:` prefix, perhaps we could use `explode`'s `$limit` parameter?

```php
return array_map(
    function ($repecId) {
        return 'RePEc:' . explode(':', $repecId, 2)[1];
    },
    $matches[0]
);
```